### DISCUSSION TITLE
GPUMD-v3.9.2

### DISCUSSION THREAD

**Discussion by brucefan1983** [brucefan1983/GPUMD#599] [Announcements]
* New features
  - `ensemble ti`: This keyword is used to set up an equilibrium thermodynamic integration integrator. It is for testing purpose and its only differece from [ti_spring keyword](https://gpumd.org/dev/gpumd/input_parameters/ensemble_ti_spring.html#kw-ensemble-ti-spring) is that the lambda value is fixed instead of changing.
  - `ensemble nphug`: This keyword sets up a Hugoniot thermostat integrator.
  - `ensemble mirror`: This keyword is employed to configure a momentum mirror shock wave simulation, where atoms are deflected by a moving momentum mirror to generate a shock wave.
  - `ensemble piston`: This keyword is employed to configure a piston shock wave simulation, where a fixed wall of atoms is displaced at a specified velocity to generate a shock wave.
  - `dump_piston`: Piston simulations commonly involve millions of atoms. Dumping all the virial and velocity data for each atom can lead to excessively large output files, making data processing cumbersome. The dump_piston command addresses this by calculating spatial thermo information during the simulation.
  - `dump_dipole`: Predicts the dipole on the fly using a TNEP model, see https://doi.org/10.1021/acs.jctc.3c01343
  - `dump_polarizability`: Predicts the polarizability on the fly using a TNEP model, see https://doi.org/10.1021/acs.jctc.3c01343
 
* Enhancements and changes
  - Changed the default values of `basis_size` for NEP training from (12, 12)  to (8, 8).
  - Improved the default regularization methods.
  - Added `stress_train.out` and `stress_test.out` during NEP training
  - Added option for `compute_shc` to calculate the SHC for all the groups in a grouping method simultaneously.

<hr /><em>This discussion was created from the release <a href='https://github.com/brucefan1983/GPUMD/releases/tag/v3.9.2'>GPUMD-v3.9.2</a>.</em>

**Comment by shdchen** [comment]
Nice new features and enhancements!

**Comment by university12138** [comment]
Hello, Mr. Fan, I want to ask if there is a file about the rebo potential function, because I am doing MoS2.

**Reply by brucefan1983** [reply]
REBO for Mos2 is not supported by recent versions of GPUMD any more. You have two options:
* Use GPUMD-v3.3.1, which has this potential, but expect less support from developers.
* Use the latest GPUMD (3.9.2) with a NEP potential model, either from others or trained by yourself. You can check this paper for a NEP mode for MoS2: 
  Tuning the Through-Plane Lattice Thermal Conductivity in van der Waals Structures through Rotational (Dis)ordering


  https://doi.org/10.1021/acsnano.3c09717

**Comment by university12138** [comment]
OK, thank Mr. Fan.
